tools/integration_tests/setup/setup.go

```diff
@@ -209,13 +209,17 @@ func ExecuteTestForFlags(flags [][]string, m *testing.M) (successCode int) {
 	return
 }
 
-func RunTests(flags [][]string, m *testing.M) {
+func ParseSetUpFlags() {
 	flag.Parse()
 
 	if !*integrationTest {
```

Comment:
For reviewer: added this here for the tests which are not using RunTests from setup